Library/Formula/openvpn.rb

```diff
@@ -25,12 +35,22 @@ def install
         "security/pam_appl.h", "pam/pam_appl.h"
     end
 
+    opts = []
+    if build.with? "pkcs11"
+        opts << "--enable-pkcs11"
+        ENV.append_path "PKG_CONFIG_PATH", "/usr/local/lib/pkgconfig"
+        ENV.append "PKCS11_HELPER_CFLAGS", `pkg-config --cflags libpkcs11-helper-1`.chomp
+        ENV.append "PKCS11_HELPER_LIBS",   `pkg-config --libs   libpkcs11-helper-1`.chomp
+    end
+
     system "./configure", "--disable-debug",
                           "--disable-dependency-tracking",
                           "--disable-silent-rules",
                           "--with-crypto-library=openssl",
                           "--prefix=#{prefix}",
-                          "--enable-password-save"
+                          "--enable-password-save",
+                          *opts
```

Comment:
Rather than tacking an `*opts` on would prefer to use an `args = %W[]` array and then:
```ruby
system "./configure", *args
```